.env.example

```
@@ -8,23 +8,23 @@ VITE_GOOGLE_CLIENT_ID=your_google_client_id_here.apps.googleusercontent.com
-# 4. Activez l'API Google Calendar
-# 
-# Service Account Details:
-# Email: planningadmin@apaddicto.iam.gserviceaccount.com
-# ID unique: 117226736084884112171
+# Configuration pour lire les disponibilités depuis Google Calendar
+# et créer des rendez-vous avec authentification
 # 
+# URL iCal publique de votre Google Calendar (pour lire les disponibilités)
+GOOGLE_CALENDAR_ICAL_URL=https://calendar.google.com/calendar/ical/doriansarry47%40gmail.com/public/basic.ics
+# Votre email Google Calendar
+GOOGLE_CALENDAR_EMAIL=doriansarry47@gmail.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GOOGLE_CALENDAR_EMAIL key should go before the GOOGLE_CALENDAR_ICAL_URL key

(UnorderedKey)